spirv_msl.hpp

```diff
@@ -156,10 +162,13 @@ class CompilerMSL : public CompilerGLSL
 	MSLResourceBinding next_metal_resource_index;
 	uint32_t stage_in_var_id = 0;
 	uint32_t stage_out_var_id = 0;
+	uint32_t stage_uniforms_var_id = 0;
 	std::string qual_pos_var_name;
 	std::string stage_in_var_name = "in";
 	std::string stage_out_var_name = "out";
+	std::string stage_uniform_var_name = "uniforms";
 	std::string sampler_name_suffix = "Smplr";
+	std::vector<std::string> reserved_names = { "kernel", "bias" };
```

Comment:
This could just be moved into the member that does the checking, just like GL backend as a static const.